Hardware/Star64.md

```diff
@@ -0,0 +1,34 @@
+---
+riscv_hardware: true
+cmake_plat: star64
+xcompiler_arg: -DRISCV64=1
+platform: Star64
+arch: RV64GBC, RV64IMAC, RV32IMAFBC
```

Comment:
Should we really list `RV32IMAFBC` here? The E24 only has M-Mode and U-Mode, so seL4 cannot run there. Same actuall for `RV64IMAC` of the S7, seL4 cannot run there, so there is ont much point to mention this here.

Reply:
I was being consistent with the HiFive Unleashed, it has a S51 core which also lacks S-Mode. I interpreted the hardware page as what each hardware platform can support, rather than what seL4 supports. Another example is the Rocketchip which has virtualisation support as "Yes", despite the kernel not being able to run in hypervisor mode on RISC-V (yet).